Task01/Task01/Program.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task01
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.ForegroundColor = ConsoleColor.Yellow;
+            Console.WriteLine("А. С. Пушкин");
+            Console.ForegroundColor = ConsoleColor.Magenta;
+            Console.WriteLine("Евгений Онегин (отрывок)");
```

Comment:
Перед текстом лучше пропустить строку
```suggestion
            Console.WriteLine("Евгений Онегин (отрывок)");
             Console.WriteLine();
```